src/System.Net.Http/src/System/Net/Http/HttpClientHandler.Net46.cs

```diff
@@ -0,0 +1,1199 @@
+            // Also note that the connection group name provided is considered an 'internal' connection group. I.e.
+            // HWR will add 'I>' after the string we provided. I.e. by default the actual connection group name looks 
+            // like '123456S>I>' or '123456U>I>' if UnsafeAuthenticatedConnectionSharing is true. Even is users use the
+            // same hashcode for their HWR connection group, they'll end up using a different one, since 'I>' is not
+            // added ('123456S>' or '123456U>').
+            if (request.Content != null)
+            {
+                webRequest = new HttpWebRequest(request.RequestUri, true, _connectionGroupName, request.Content.CopyTo);
+            }
+            else
+            {
+                webRequest = new HttpWebRequest(request.RequestUri, true, _connectionGroupName, null);
+            }
+
+            if (Logging.On) Logging.Associate(Logging.Http, request, webRequest);
+
+            webRequest.Method = request.Method.Method;
+            webRequest.ProtocolVersion = request.Version;
+
+            SetDefaultOptions(webRequest);
+            SetConnectionOptions(webRequest, request);
+            SetServicePointOptions(webRequest, request);
+            SetRequestHeaders(webRequest, request);
+            SetContentHeaders(webRequest, request);
+
+            // New properties for this OOB HttpClientHandler.
+            webRequest.ServicePoint.ConnectionLimit = _maxConnectionsPerServer;
+            webRequest.MaximumResponseHeadersLength = _maxResponseHeadersLength;
+            if ((ClientCertificateOptions == ClientCertificateOption.Manual)
+                && (_clientCertificates != null) && (_clientCertificates.Count > 0))
+            {
+                webRequest.ClientCertificates = _clientCertificates;
+            }
+
+            if (_serverCertificateCustomValidationCallback != null)
+            {
+                webRequest.ServerCertificateValidationCallback = ServerCertificateValidationCallback;
+            }
+
+            if (_defaultProxyCredentials != null && webRequest.Proxy != null)
+            {
+                webRequest.Proxy.Credentials = _defaultProxyCredentials;
+            }
+
+            // For Extensibility
+            InitializeWebRequest(request, webRequest);
+
+            return webRequest;
+        }
+
+        // Used to map the ServerCertificateCustomValidationCallback which uses Func<T> to the
+        // HttpWebRequest based RemoteCertificateValidationCallback delegate type.
+        private bool ServerCertificateValidationCallback(
+            object sender,
+            X509Certificate certificate,
+            X509Chain chain,
+            SslPolicyErrors sslPolicyErrors)
+        {
+            return _serverCertificateCustomValidationCallback(
+                null, // TODO: How to map sender (which is really an HttpWebRequest) to an HttpRequestMessage?
```

Comment:
For now, we're using only one issue, #11100 as we continue to iterate on this even beyond this PR. So, I'll reference that in the TODO.